include/osrm/coordinate.hpp

```diff
@@ -1,72 +1,11 @@
-/*
+#ifndef GLOBAL_COORDINATE_HPP
+#define GLOBAL_COORDINATE_HPP
 
```

Comment:
Would also be fixed with folder in the header guard instead of using `global_coordinate_hpp`.